.github/dependabot.yml

```
@@ -0,0 +1,69 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "pip" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    milestone: 2
+    target-branch: "master"
+    versioning-strategy: increase-if-necessary
+    # Labels on pull requests for version updates only
+    labels:
+      - "Configs"
+      - "Duplicate"
+      - "Python Lang"
+      - "Python Repo"
+    groups:
+      production-dependencies:
+        dependency-type: "production"
+        exclude-patterns:
+          - "setuptools*"
+      development-dependencies:
+        dependency-type: "development"
+    allow: 
+      - dependency-name: "setuptools"
+        dependency-type: "production"
+      - dependency-name: "pip"
+        dependency-type: "direct"
+      - dependency-name: "wheel"
+        dependency-type: "production"
+      - dependency-name: "build"
+        dependency-type: "production"
+      - dependency-name: "argparse"
+        dependency-type: "direct"
+      - dependency-name: "six"
+        dependency-type: "direct"
+      - dependency-name: "tox"
+        dependency-type: "development"
+      - dependency-name: "virtualenv"
+        dependency-type: "development"
+    assignees:
+      - "reactive-firewall"
+    commit-message:
+      prefix: "[HOTFIX] "
+      include: "scope"
+    schedule:
+      interval: "weekly"
+      day: "tuesday"
+  - package-ecosystem: "github-actions" # See documentation for possible values
+    directory: ".github/workflows/" # Location of package manifests
+    milestone: 2
+    target-branch: "master"
+    # Labels on pull requests for version updates only
+    labels:
+      - "Configs"
+      - "Duplicate"
```

@coderabbitci Looks like another case of the [misleading label issue](https://github.com/reactive-firewall/python-repo/pull/66/files#r1746229166) we already discussed. Let's fix it with a similar resolution:

```suggestion
     - "Version Update"
```